packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -279,7 +277,7 @@ class AndroidDevice extends Device {
     if (!await _checkForSupportedAdbVersion() || !await _checkForSupportedAndroidVersion())
       return false;
 
-    final Status status = logger.startProgress('Installing ${fs.path.relative(apk.file.path)}...', expectSlowOperation: true);
+    final Status status = logger.startProgress('Installing ${fs.path.relative(apk.file.path)}...', timeout: slowOperation);
```

Comment:
Where is `slowOperation` defined?

Reply:
Nevermind - found it.

Reply:
Same place as `startProgress` (logger.dart).